src/sentry/static/sentry/app/components/releaseStats.jsx

```diff
@@ -22,16 +22,18 @@ const ReleaseStats = React.createClass({
       return null;
     }
     return (
-      <div className="release-info">
-        <div><b>{commitCount}{t(' commits by ')}{authorCount}{t(' authors')}</b></div>
-        {release.authors.map(author => {
-          return (
-            <span className="assignee-selector tip"
-                 title={author.name + ' ' + author.email}>
-              <Avatar user={author}/>
-            </span>
-          );
-        })}
+      <div className="release-stats">
+        <h6>{commitCount}{t(' commits by ')}{authorCount}{t(' authors')}</h6>
```

Comment:
This isn't a proper translatable string, and if we commit it as-is it will get pulled into Transifex for translation.

You want:

```javascript
import {tct} from '../locale';
// ...

tct('[commitCount] commits by [authorCount] authors', {
  [commitCount],
  [authorCount]
});
```